examples/nlg_server/nlg_server.py

```diff
@@ -70,11 +73,7 @@ def nlg():
     cmdline_args = arg_parser.parse_args()
 
     domain = Domain.load(cmdline_args.domain)
```

Comment:
this isn't a good idea: `domain` is used as a global variable in L63 - but if main is not run, that variable does not exist